src/MICore/LaunchOptions.cs

```diff
@@ -405,6 +405,13 @@ public sealed class UnixShellPortLaunchOptions : LaunchOptions
         public string StartRemoteDebuggerCommand { get; private set; }
         public Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.IDebugUnixShellPort UnixPort { get; private set; }
 
+        public static bool LastDebuggerLaunchSuccessful = false;
+
+        private static string GetClrDbgUrl = "https://raw.githubusercontent.com/Microsoft/MIEngine/dev/rajkumar42/updatescript/scripts/GetClrDbg.sh";
+        private static string DebugerInstallationDirectory = "~/.vs-debugger";
+        private static string DebuggerFirstLaunchCommand = "mkdir -p {0} && wget -q {1} -O {0}/GetClrDbg.sh && chmod +x {0}/GetClrDbg.sh && {0}/GetClrDbg.sh -v latest -l {0}/latest -d -u";
```

Comment:
If you use '-q' with wget it will not output an error if there is no internet connectivity.